packages/jetpack-blocks/src/blocks/wordads/editor.scss

```diff
@@ -30,14 +30,22 @@
 	.components-placeholder {
 		flex-grow: 2;
 	}
+
+	.components-toggle-control__label {
+		line-height: 1.4em;
+	}
 }
 
 .jetpack-wordads-leaderboard .components-placeholder {
 	min-height: 90px;
 }
 
 .jetpack-wordads-mobile_leaderboard .components-placeholder {
```

Comment:
Looks like the same block as above, shall we merge these rules?

Reply:
I ended up putting it back to 72 px since the control is not outside the placeholder.